src/common/bool.js

```diff
@@ -0,0 +1,3 @@
+import Joi from 'joi';
+
+export default description => Joi.boolean().description(description);
```

Comment:
It was so the name could match specifically to each route since two routes the same schema.  After talking this over, I want to change it, but I want to wait until after we aren't block the public API team.